app/services/intercom.js

```diff
@@ -9,5 +9,29 @@ export default Ember.Service.extend({
 
   update(params) {
     Intercom('update', params);
+  },
+
+  shutdown() {
+    Intercom('shutdown');
+  },
+
+  hide() {
+    Intercom('hide');
+  },
+
+  show() {
+    Intercom('show');
+  },
+
+  open() {
```

Comment:
What about calling this `showMessages`, to match the signature used by the Intercom API, likewise `showNewMessage` rather than `newMessage` below?